Testscripts/Windows/AddHardDisk.ps1

```diff
@@ -342,7 +342,8 @@ function Main {
 
     $vmGen = Get-VMGeneration $VMName $HvServer
     if ($vmGen -ne 1) {
-        throw "VHD is not supported by Gen 2 VMs"
+        Write-LogInfo "Generation 2 VM does not support vhd disk, please skip this case in the test script"
```

Comment:
you don't judge current disk is vhd not vhdx?

Reply:
By default, this AddHardDisk.ps1 file is used for add VHD storage,  if add vhdx, it will use AddVhdxHardDisk.ps1, if add check maybe better, will double check it.

Reply:
I think we should check both cases as we will have more usage of Gen2 images.